src/libraries/System.Data.Common/src/System/Data/Common/DbConnectionStringBuilder.cs

```diff
@@ -12,10 +12,8 @@
 
 namespace System.Data.Common
 {
-    // This coarse suppression silences all RequiresUnreferencedCode warnings in the class.
-    // https://github.com/mono/linker/issues/2136 tracks making it possible to add more granular suppressions at the member level, and with a different warning code.
-    [UnconditionalSuppressMessage("ReflectionAnalysis", "IL2026:RequiresUnreferencedCode",
-        Justification = "The use of GetType preserves members with RequiresUnreferencedCode, but the GetType callsites either "
+    [UnconditionalSuppressMessage("ReflectionAnalysis", "IL2113:RequiresUnreferencedCode",
```

Comment:
(nit) - `"IL2113:RequiresUnreferencedCode"` should be `"IL2113:RequiresDynamicallyAccessedMembers" (or whatever short name we decided to give these new warnings).

This is repeated elsewhere for IL2112 in this file, and other files.

Reply:
My thinking was that RequiresUnreferencedCode here was about the DAM-on-type requiring methods annotated with RequiresUnreferencedCode. I renamed it to ReflectionToRequiresUnreferencedCode. Just note that some instances of IL2026 also represent ReflectionToRequiresUnreferencedCode, for instance:

```csharp
typeof(Foo).GetMethod("RUC"); // IL2026

class Foo {
	[RUC("RUC")]
	public static void RUC() {}
}
```